### DISCUSSION TITLE
enhancement: add padding-y prop to the container component

### DISCUSSION THREAD

**Discussion by anuraglol** [heroui-inc/heroui#261] [Ideas]
Currently, the gap prop of the Container component is only about padding of x-axis, but there should be another prop for padding of the y-axis

#### Solution I would like to recommend:

A new prop probably named `py` which controls the paddingY property of the component

**Comment by anuraglol** [comment]
btw, with your consent, can I work on this?

**Comment by jrgarciadev** [comment]
Hey @kr-anurag thanks for the recommendation but, why don't you use `css={{ py: 'PADDING' }}`?
